src/check.js

```diff
@@ -0,0 +1,27 @@
+export default function elementChanges({
+  checkbox, input, element, li, ToDoArray,
+}) {
```

Comment:
- [ ] We are trying to write code using ES6 syntax. Kindly convert this to an ES6 arrow function.
- [ ] Look at the image below. There is something wrong with your app when I check the box. It appears to clear the localStorage. Kindly fix this behaviour

![meq-check](https://user-images.githubusercontent.com/36542986/144607674-3bc797bf-65ca-4447-a642-a14e592ea5ec.jpg)